target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_avm_fritz5490.dtsi

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "vr9_avm_fritzxx90.dtsi"
+
+/ {
+	compatible = "avm,fritz5490", "lantiq,xway", "lantiq,vr9";
+	model = "AVM FRITZ!Box 5490/5491";
+};
+
+&aliases {
+	led-dsl = &led_info_green;
+	led-internet = &led_internet;
+	led-wifi = &led_wifi;
+};
+
+&leds {
+	led_fiber: fiber {
+		label = "green:fiber";
+		gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
+	};
+
+	led_wifi: wifi {
+		label = "green:wlan";
+		gpios = <&gpio 36 GPIO_ACTIVE_LOW>;
+	};
+
+	led_internet: internet {
+		label = "green:internet";
+		gpios = <&gpio 35 GPIO_ACTIVE_LOW>;
+	};
+};
+
```

Comment:
The pimux for mdio is missing:
```
&gswip {
	pinctrl-0 = <&mdio_pins>;
	pinctrl-names = "default";
};

```
It probably works because the bootloader sets the appropriate values.